### ISSUE TITLE
Privacy Policy: Console Error

### ISSUE THREAD

**Issue by romatitan** [titan-cebu/BNT-Legal#9]
![image](https://cloud.githubusercontent.com/assets/12888048/9377046/506b08da-4747-11e5-82ea-9f67f097def9.png)

**Comment by romatitan** [comment]
![image](https://cloud.githubusercontent.com/assets/12888048/9513533/b148c6fc-4cc3-11e5-8ead-5b9d0f8f7b31.png)

**Comment by christiancebudigital** [comment]
Please clear your cache.. no error found on our end...

**Comment by romatitan** [comment]
I've cleared the cache and check again the console.

Chrome - console error: Falied to load resource: net::ERR_BLOCKED_BY_CLIENT
The error still occurs at my end, but I've tried to check with other PC and chrome-browser and there's no error occurs.
Result: 1:3
For chrome browser its won't fix.

But still a console error occurs in IE browsers.
IE - console error: SCRIPTS: Access is denied.
Result: 3:3
![image](https://cloud.githubusercontent.com/assets/12888048/9538579/9485921a-4d78-11e5-97ba-15ef7f84076c.png)
